.github/workflows/ci.yml

```diff
@@ -50,12 +50,12 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-    - name: Install dependencies
-      run: |
-        sudo apt install verilator libelf-dev
-        pip install -r python-requirements.txt
+    - name: Install Nix
+      uses: cachix/install-nix-action@v24
```

Comment:
To enable cache globally for all nix commands, you can do something similar to https://github.com/lowRISC/lowrisc-nix/blob/34f1d472d6066a42c684416f5e9e755d25ec6270/.github/workflows/ci.yml#L52-L54 here.